### ISSUE TITLE
Request: 768p output

### ISSUE THREAD

**Issue by segacollection** [ramapcsx2/gbs-control#319]
It would be fantastic to have native 768p output for my Vewlix cabinet... is this possible?

**Comment by segacollection** [comment]
Is this easy/possible to implement?

**Comment by sztojanov** [comment]
please! please DO the 1024x768 output! I would like to use the GBS with my projector which has ONLY 768p input acceptable. thanks!

**Comment by RedAnon** [comment]
Yes this would be nice for some old plasma TVs, PC monitors etc.

**Comment by sztojanov** [comment]
and for my retro 4:3 projector, which has 1024x768 input mode!

**Comment by mpalpha** [comment]
[https://github.com/mpalpha/gbs-control
https://gist.github.com/mpalpha/a0d75432c6c8882557be139bde09471e](https://github.com/mpalpha/gbs-control-odv/releases/tag/1.0.24)
